component/portal/src/test/java/org/exoplatform/portal/rest/UserFieldValidatorTest.java

```diff
@@ -59,6 +59,18 @@ public void testValidatePersonalName() {
     assertNull(fieldValidator.validate(Locale.ENGLISH, "aaa'bb"));
   }
 
+  @Test
+  public void testValidateEmail() {
+    UserFieldValidator fieldValidator = new UserFieldValidator("email", false, false);
+    assertNotNull(fieldValidator.validate(Locale.ENGLISH, ""));
+    assertNotNull(fieldValidator.validate(Locale.ENGLISH, "WRONG_FORMAT"));
+    assertNotNull(fieldValidator.validate(Locale.ENGLISH, "WRONG_FORMAT@"));
+    assertNotNull(fieldValidator.validate(Locale.ENGLISH, "WRONG_FORMAT@test"));
+    assertNotNull(fieldValidator.validate(Locale.ENGLISH, "@test.com"));
+    assertNull(fieldValidator.validate(Locale.ENGLISH, "GOOD_FORMAT@test.com"));
+    assertNull(fieldValidator.validate(Locale.ENGLISH, "GOOD_FORMAT@test.test"));
```

Comment:
how about user.name@domain.co.uk ?

Reply:
works as well

Reply:
then an UT assertion could document it ;-)